messages-cosmos.proto

```diff
@@ -0,0 +1,62 @@
+syntax = "proto2";
+
+// Sugar for easier handling in Java
+option java_package = "com.keepkey.deviceprotocol";
+option java_outer_classname = "KeepKeyMessageCosmos";
+
+import "types.proto";
+
+/**
+ * Request: Address at the specified index
+ * @start
+ * @next CosmosAddress
+ */
+message CosmosGetAddress {
+    repeated uint32 address_n = 1;  // BIP-32 path. For compatibility with other wallets, must be m/44'/148'/index'
+    optional bool show_display = 2; // optionally show on display before sending the result
+}
+
+/**
+ * Response: Address for the given index
+ * @end
+ */
+message CosmosAddress {
+    optional string address = 1;    // Address Bech32 encoded with 'cosmos' prefix
+}
+
+/**
+ * Request: ask device to sign Cosmos transaction
+ * @start
+ * @next CosmosSignedTx
+ */
+message CosmosSignTx {
+    repeated uint32 address_n = 1;          // BIP-32 path. For compatibility with other wallets, must be m/44'/118'/index'
+    optional uint64 account_number = 2;
+    optional string chain_id = 3;
+    optional uint32 fee_amount = 4;
+    optional uint32 gas = 5;
+    optional string memo = 6;
+    optional uint64 sequence = 7;
```

Comment:
```suggestion
    optional uint64 sequence = 7 [jstype = JS_STRING];
```

Reply:
could you explain what this does?

Reply:
When we generate the javascript/typescript bindings, this specifies the type used to represent this field. Since `number` in js is really a float64, it can't precisely represent integers that are larger than will fit in the mantissa... but we can represent it with a string.

see: https://github.com/protocolbuffers/protobuf/issues/3666

Reply:
This shouldn't affect the C/Python bindings.